Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,15 @@
+class Pacapt < Formula
+
+  homepage "https://github.com/icy/pacapt" 
+  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4", :tag => "v2.0.2"
```

Comment:
No there are no submodules. At first I tried to use directly a tarball, 
but it did notwork. The compilation script checks the git for a version 
or something like that. I do not know exactly, and do not really 
underestand why they do that. ;)

On 03/30/2015 11:42 PM, Dominyk Tiller wrote:
>
> In Library/Formula/pacapt.rb 
> <https://github.com/Homebrew/homebrew/pull/38188#discussion_r27434938>:
>
> > @@ -0,0 +1,15 @@
> > +class Pacapt < Formula
> > +
> > +  homepage "https://github.com/icy/pacapt"
> > +  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4", :tag => "v2.0.2"
>
> Depends on |git|? In which regards? It uses it for submodules or?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Homebrew/homebrew/pull/38188/files#r27434938>.
>